cmd/validator/slashing-protection/export.go

```diff
@@ -1,4 +1,4 @@
-package slashingprotection
+package slashingprotectioncmd
```

Comment:
This is a really long package name. Considering packages like this are only imported by the command that uses them, maybe we could call them something simple and generic like `subcmd`.